src/comm/client/client-util.js

```diff
@@ -9,7 +10,80 @@
 'use strict';
 const logger = require('../util.js').getLogger('client-util.js');
 let processes  = {}; // {pid:{obj, promise}}
+let txUpdateTime = 1000;
+const kafka = require('kafka-node');
+const listener_config = require('../../listener/listener-config.json');
```

Comment:
Since Kafka mode is not necessary for all kinds of the tests,  these requirements may need to move into the related function.